src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -54,6 +54,11 @@ export type ConnectWallet = {
 }
 
 export type Portfolio = {
+  /**
+   * The app can run account discovery taking a lot of time and slowing down the portfolio initialization,
+   * we use this key to gracefully handle this state in any part of the app
+   */
+  isAccountLoading: boolean
```

Comment:
q: can we leverage RTK query slice instead? 

see e.g:

https://github.com/shapeshift/web/blob/d72c8d499e3898caffbc695e18efa56a0eeb701f/src/state/slices/txHistorySlice/selectors.ts#L34-L38

Reply:
After testing, I would say no:

- Pending queries starts 6 secs later than the actual implementation, resulting in a wrong UI first
- Pending queries are triggered absolutely every time we forceRefresh any accounts, resulting in way more loads than needed even if we already have the accounts in the cache

While it might not be the best from an engineering perspective, the actual implementation looks better to me from a UX point of view

Reply:
Discussed this one out, agreed to keep the current implementation for now